synaptic_reconstruction/inference/vesicles.py

```diff
@@ -98,7 +99,7 @@ def segment_vesicles(
     tiling: Optional[Dict[str, Dict[str, int]]] = None,
     min_size: int = 500,
     verbose: bool = True,
-    distance_based_segmentation: bool = False,
+    distance_based_segmentation: bool = True,
```

Comment:
This got set to `False` accidentally, I think. I checked again and I think it reduces the false merge errors, so I set the default to `True`.